docs/src/pages/components/tooltips/tooltips.md

```diff
@@ -123,8 +123,8 @@ On mobile, the tooltip is displayed when the user longpresses the element and hi
 
 (WAI-ARIA: https://www.w3.org/TR/wai-aria-practices/#tooltip)
 
-By default the tooltip only labels its child element.
-This is notably different from `title` which can either label **or** describe its child depending on whether the child is already labelled.
+By default, the tooltip only labels its child element.
+This is notably different from `title` which can either label **or** describe its child depending on whether the child is already labeled.
```

Comment:
en-US

Reply:
Tricky. The corresponding aria attribute has a double-"l". Is a consistent language here more important than using the spelling consistent with the aria attribute? I wouldn't change this everywhere but when talking about a11y I would prefer `labelled` so that people don't accidentally write `aria-labeledby`.

Both are correct as far as I know. One is just preferred in the US.

/cc @mbrookes